components/Faq/FaqQandA.tsx

```diff
@@ -8,19 +8,17 @@ type faqQandAProps = {
 }
 
 export const FaqQandA = ({ question, answer }: faqQandAProps) => {
-  const [open, setOpen] = useState(false)
-
-  useEffect(() => {}, [open])
+  const [isOpen, setOpen] = useState(false)
 
   return (
     <>
-      <a
-        onClick={() => setOpen(!open)}
-        aria-controls="example-collapse-text"
-        aria-expanded={open}
+      <div
+        onClick={() => setOpen(!isOpen)}
+        aria-controls="collapse-text"
+        aria-expanded={isOpen}
       >
```

Comment:
hey the url already has the question in the url! :)

Reply:
Hey so do you know want to add a GH tix for this? When I collapse the question the question url still remains in the url. I think it may need some extra tweaking and would make a good separate ticket.